scripts/run-cypress-test-docker

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+if bash +x -o nounset -c "$(aws ecr get-login --no-include-email --region "us-west-2" --registry-ids "${AWS_ACCOUNT_ID}")" ; then
+  docker-compose pull db_migrate easi
+
+  docker-compose build --parallel easilocal cypress
+
+  docker-compose up -d db
+  sleep 5
+  docker-compose up --exit-code-from db_migrate db_migrate
+  docker-compose up -d easi easilocal
+  docker-compose up --exit-code-from cypress cypress
+  echo "done!"
```

Comment:
```suggestion
  echo 'done!'
```
I admit this is a small nit but `!` can do odd things with some shell settings even when double quoted

Reply:
Good point. I can simplify it even more and remove the `!`.